### ISSUE TITLE
Not loading

### ISSUE THREAD

**Issue by R5tl** [Shivelight/pOOBs4-buildroot#9]
I burned the image on a few different SDcard (class 10) and tried different usb cable but even if I can connect to the Orange pi zero wifi Poobs4, the page doesn't load nor on the ps4 nor on my phone (I tried to see) it get stuck. 
I can't access the board from m phone by ssh either.
Is there something i'm missing ?

**Comment by Shivelight** [comment]
Orange Pi Zero wifi module XR819 is known to have problematic firmware. I also faced the same issue, but it rarely occurred. 
It would be easier to debug if you have USB TTL serial cable. But if you don't have any, you can try to SSH via a LAN cable. 

Alternatively, you can try dumping `dmesg` by creating a new script in `/etc/init.d/`. For example `/etc/init.d/S99dumpdmesg`:

```sh
#!/bin/sh

dmesg > /root/dmesg.txt
exit 0
```

Then you can pull the dump from the SD card.

By the way, I recommend dongles with RTL8188 or MT7601 chipset, much better and very cheap.

**Comment by R5tl** [comment]
Thank you for the quick answer.
I could SSH via LAN cable but what I can do there is very limited. Not sure how to make a scrip as it seems there isn't a test editor in this image.

**Comment by Shivelight** [comment]
Great! Now, you don't need the script, just run 
`dmesg > /root/dmesg.txt` and download the log via SFTP. You can upload it here so I can take a look.

**Comment by R5tl** [comment]
[dmesg.txt](https://github.com/Shivelight/pOOBs4-buildroot/files/8004058/dmesg.txt)
Thanks to you I could get the file. Hope it's helpfull.

**Comment by Shivelight** [comment]
Is the issue persist? Looking at the log, it seems like everything is normal. I can't reproduce it on my system either.

When the board is on, login to SSH via LAN and try restarting `hostapd` by executing `/etc/init.d/S41hostapd restart`. Make sure your device gets an IP, try if you can ping `10.0.0.1`.

**Comment by saivignesh** [comment]
At first I couldn't connect via SSH or sftp, but then I connected to the pOOBs4 wifi on my computer and then I could connect via both SSH and stfp. Especially sftp was helpful to transfer files from and to the var/www/html directory since I was trying to load a different exploit. But I am unsure how to load another exploit. Do I just save the html file for host website like karo218.ir  in the var/www/html directory or what ?

**Comment by R5tl** [comment]
Now it's working. I can even access the root by SSH on 10.0.0.1
I tried so many times and now without changing anything, out of the blue it worked.

**Comment by Shivelight** [comment]
@saivignesh I wrote a short guide here https://github.com/Shivelight/pOOBs4-buildroot#bring-your-own-exploit-host. Please open a new issue if you still need help.

**Comment by Shivelight** [comment]
> Now it's working. I can even access the root by SSH on 10.0.0.1 I tried so many times and now without changing anything, out of the blue it worked.

Nice. Although, I still recommend a USB dongle to avoid issues. The wifi module can crash on boot too!

**Comment by R5tl** [comment]
Thank you @Shivelight 
Would the USB dongle be plug and play with this build our do I need to add the rtl8188 drivers ?

**Comment by Shivelight** [comment]
Yes. I can upload a pre-release image if you have trouble building the image. Or, you can always try the dev build that you can find in the actions tab, e.g., https://github.com/Shivelight/pOOBs4-buildroot/actions/runs/1784236192

**Comment by R5tl** [comment]
I'm a noob with those boards. I couldn't build the image myself from your repo. I think I don't understand the path system and BR2_EXTERNAL isn't working on my PC.
If your very kind offer to make a build with the drivers, I'll get my hands on a WiFi dongle as the OPi zero module isn't trust worthy.

**Comment by Shivelight** [comment]
Feel free to ask when your dongle arrives.
